src/user-guide/running-workflows/scheduler-log-files.rst

```diff
@@ -23,11 +23,29 @@ The information logged here includes:
 
    Workflow log files are primarily intended for human eyes. If you need
    to have an external system to monitor workflow events automatically, use:
-   
+
    * Event hooks (see :cylc:conf:`flow.cylc[scheduler][events]` and
      :cylc:conf:`flow.cylc[runtime][<namespace>][events]`).
    * The GraphQL interface (can be accessed via GraphiQL in the Cylc GUI).
    * The sqlite *workflow run database*
      (see :ref:`Workflow Run Databases`)
-     
+
    Rather than parse the log files.
+
+
+Workflow Configuration Logs
+---------------------------
+
+When a workflow starts it records configurations in the
+:term:`workflow log directory` (``$HOME/cylc-run/WORKFLOW-NAME/log/flow-config/``).
+
+In both cases any templating is expanded:
+
+- ``flow-processed.cylc`` - the current workflow configuration
+  with duplicate sections unmerged.
+- ``<datetime-stamp>-<start/restart/reload>`` - the config at each time the workflow
+  was started, restarted, or reloaded; with duplicate sections merged.
+
+.. note::
+
+   Look at these files to see the result of templating in the workflow config.
```

Comment:
Was this meant to be included in this PR?

Reply:
No.